db/keyval/etcdv3/plugin_impl.go

```diff
@@ -33,6 +34,7 @@ const (
 
 // Plugin implements Plugin interface therefore can be loaded with other plugins
 type Plugin struct {
+	Transport      *adapters.TransportAggregator
```

Comment:
skeleton uses it to create transport, but the aggregator cannot be injected directly into it because the skeleton is not initialized until etcd/redis Init(). So if we do not want to have the skeleton in flavor...